azure-functions/acbs-function/.prettierrc.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Could we instead remove all the `.prettierignore` and `.prettierrc.json` files from the individual sub-projects, and just have them all fallback to the versions in the root of the main `dtfs2` project? I think this would happen automatically if these are removed, so wouldn't require any extra config. This would ensure our rules are always consistent across projects.

Reply:
Good shout, have removed.
Have also removed `.eslintignore` and will look into removing `.eslintrc.js` (where duplicate only)